go.mod

```diff
@@ -44,17 +44,16 @@ require (
 	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
 	github.com/googleapis/gax-go v2.0.2+incompatible // indirect
 	github.com/gorhill/cronexpr v0.0.0-20180427100037-88b0669f7d75 // indirect
-	github.com/gorilla/websocket v1.4.0 // indirect
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0 // indirect
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0 // indirect
 	github.com/hashicorp/consul v1.4.2 // indirect
 	github.com/hashicorp/go-gcp-common v0.0.0-20180425173946-763e39302965 // indirect
 	github.com/hashicorp/go-plugin v0.0.0-20190220160451-3f118e8ee104 // indirect
 	github.com/hashicorp/go-retryablehttp v0.5.2 // indirect
-	github.com/hashicorp/go-rootcerts v1.0.0 // indirect
 	github.com/hashicorp/go-version v1.1.0 // indirect
 	github.com/hashicorp/golang-lru v0.5.1 // indirect
-	github.com/hashicorp/nomad v0.9.0-beta3.0.20190228221024-82d37b99dff9
+	github.com/hashicorp/nomad v0.11.3
```

Comment:
Upgraded the Nomad API, all other changes are indirect

Reply:
Have you tried if the Nomad clients can communicate properly?

Assuming the Nomad clients are on 0.9, this might cause breaking changes from that perspective, but I guess most of the time Nomad will try to be backward compatible.

Reply:
This is a good point, will keep in mind. I have tested in staging and it's able to communicate with the Nomad clients thus far.

Reply:
Well this is inevitable, we would have to upgrade at some point.
So far Nomad seems fairly backward compatible so should be fine.